File: C/Exercises/1ª Lista de Exercícios/Exercício04.c
#include <stdio.h>

int main()
{

    int dias, horas, minutos, segundos, total;

    printf("Digite o total de dias: ");
    scanf("%d", &dias);

    printf("Digite o total de horas: ");
    scanf("%d", &horas);

    printf("Digite o total de minutos: ");
    scanf("%d", &minutos);

    printf("Digite o total de segundos: ");
    scanf("%d", &segundos);

    total = segundos + (minutos * 60) + (horas * 3600) + (dias * 24 * 3600);

    printf("Total de segundos: %d\n", total);

}

File: Java/1ª Lista de Exercícios (1S)/Exercício04/Exercício04.java
import java.util.Scanner;

public class Exercício04 {
	public static void main(String args[]){
		Scanner scan = new Scanner(System.in);
		
		System.out.print("Dias: ");
		int dias = scan.nextInt();

		System.out.print("Horas: ");
		int horas = scan.nextInt();

		System.out.print("Minutos: ");
		int minutos = scan.nextInt();
		
		System.out.print("Segundos: ");
		int segundos = scan.nextInt();

		int total = (dias*24*60*60) + (horas*60*60) + (minutos*60) + segundos;

		System.out.println("Total de segundos: " + total);

		scan.close();

	}
}
